runners_manager/runner/RunnerManager.py

```diff
@@ -107,10 +112,11 @@ def delete_runner(self, runner: Runner):
         except APIException:
             logger.info(f'APIException catch, for runner: {str(runner)}')
 
-    def next_runner_name(self):
-        name = f'{self.runner_counter}'
-        self.runner_counter += 1
-        return name
+    def generate_runner_name(self, vm_type: VmType):
+        return self.runner_name_format.format(index=self.runner_counter,
+                                              organization=self.github_organization,
+                                              flavor=vm_type.flavor,
+                                              image=vm_type.image).replace(' ', '-')
```

Comment:
Not sure what case you encountered to need this replace, or if it's just a safety.

Reply:
The name we use for images contain whitespaces :/

Reply:
Are you talking about the name of the image that we give on the github side? or the image in openstack?

Meaning:
* On github I thought we called the image `centos7` or `centos` or `centos-7` -> Where I see no space here
* on openstack we don't have much control there and indeed I think some contain whitespace, but those are not the image name that we put on the runner name AFAIK